docs/migration_guide.rst

```diff
@@ -1,6 +1,17 @@
 Migration guide
 ===============
 
+******************************
+0.22.0 Migration guide
+******************************
+
+Version 0.22.0 of **starknet.py**
+
+0.22.0 Breaking changes
+-----------------------
+
+1. Support for Goerli has been removed
```

Comment:
Let's also include in the breaking changes that `StarknetChainId.SEPOLIA_TESTNET` has been renamed to `StarknetChainId.SEPOLIA`.